en/docs/get-started/api-manager-quick-start-guide.md

```diff
@@ -21,17 +27,22 @@ This section is a step-by-step guide to create, publish, and invoke an API using
 
 Follow the instructions below to create and publish an API via the Publisher Portal of WSO2 API-M.
 
-1. Navigate to the Publisher Portal [https://localhost:9443/publisher](https://localhost:9443/publisher) and sign in with **`admin/admin`** as the credentials (The below home screen will be displayed).
+1. Navigate to the Publisher Portal.
+   
+     [https://localhost:9443/publisher](https://localhost:9443/publisher)
+     
+2. Sign in with **`admin/admin`** as the credentials.
                                                  
-    [![Publisher portal home page]({{base_path}}/assets/img/get_started/api-publisher-home.png)]({{base_path}}/assets/img/get_started/api-publisher-home.png)
+     [![Publisher portal home page]({{base_path}}/assets/img/get_started/api-publisher-home.png)]({{base_path}}/assets/img/get_started/api-publisher-home.png)
 
-2. Create an API.
+3. Create an API.
 
      Let's use a mock REST service to create a REST API from scratch.
  
-     A mock service with a JSON response `{"hello": "world"}`  is provided by default when you use the service URL as (`http://run.mocky.io/v2/5185415ba171ea3a00704eed`) that appears in the [https://designer.mocky.io/](https://designer.mocky.io/) mock service. Note that we are using the HTTP protocol instead of HTTPS.
+     A mock service with a JSON response `{"hello": "world"}`  is provided by default when you use the service URL as `http://run.mocky.io/v2/5185415ba171ea3a00704eed`, which appears in the [https://designer.mocky.io/](https://designer.mocky.io/) mock service. Let's use the HTTP protocol instead of the HTTPS protocol for this guide.
```

Comment:
```suggestion
     A mock service with a JSON response `{"hello": "world"}` is provided by default when you use the service URL as `http://run.mocky.io/v2/5185415ba171ea3a00704eed`, which appears in the [https://designer.mocky.io/](https://designer.mocky.io/) mock service. Let's use the HTTP protocol instead of the HTTPS protocol for this guide.
```